apps/cow-tools/next.config.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-env node */
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { withSentryConfig } = require("@sentry/nextjs");
+
+/** @type {import('next').NextConfig} */
+const moduleExports = {
```

Comment:
@ribeirojose I used the same config from balancer-tools. Why was Hound caught here but not there? it's because these files were added there before Hound?

Reply:
Not sure, but you can ignore it